.circleci/config.yml

```diff
@@ -14,6 +14,7 @@ workflows:
               only: /^v.*/
 
       - architect/push-to-docker:
+          context: "architect"
```

Comment:
@xh3b4sd like this.

Reply:
So what is the rule? Is it for all push jobs?

Reply:
Jobs that need env variables need the context. I updated examples in docs in architect-orb.